jgrapht-core/src/main/java/org/jgrapht/GraphPath.java

```diff
@@ -95,6 +124,15 @@
      * @return the weight of the path
      */
     double getWeight();
+
+    /**
+     * Returns the length of the path, measured in the number of vertices.
+     * @return the length of the path, measured in the number of vertices
+     */
+    default int getLength(){
```

Comment:
I agree, that method should have returned the number of edges, not the number of vertices. If desired I can add a method like getVertexCount or something.